index.html

```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
 
-<head>
```

Comment:
Dont remove the head tag ... update it with the head tag and then create a new pr

Reply:
Lan head told me that in place of forza csgo will take place